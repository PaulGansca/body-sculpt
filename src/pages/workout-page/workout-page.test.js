import { shallow } from 'enzyme';
import React from 'react';
import { Provider } from 'react-redux';
import WorkoutPage from './workout-page';
import WorkoutPageContainer from './workout-page.container';

let wrapper;

describe('<WorkoutPage />', () => {
    it("container renders workout based on id", () => {
        const mockProps = {
            match: {params: {workoutId: 1}},
            isUserLoading: true,
            isWorkoutLoading: true,
            getState: jest.fn(),
            subscribe: jest.fn(),
            dispatch: jest.fn(),
            workouts: [{id: 1}],
        }
        wrapper = shallow(
                    <Provider store={mockProps}>
                        <WorkoutPageContainer {...mockProps} />
                    </Provider>)
        expect(wrapper).toMatchSnapshot();
    })
    it("expect to render workout page", () => {
        wrapper = shallow(<WorkoutPage />)
        expect(wrapper).toMatchSnapshot();
    })
});