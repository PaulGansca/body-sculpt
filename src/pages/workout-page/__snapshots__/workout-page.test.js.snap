// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WorkoutPage /> container renders workout based on id 1`] = `ShallowWrapper {}`;

exports[`<WorkoutPage /> expect to render workout page 1`] = `ShallowWrapper {}`;
